docs/reporting.rst

```diff
@@ -74,48 +74,38 @@ The following are listed as the data sources currently available for reports:
 All PMHC Program Types
 ----------------------
 
-The All PMHC Program Types data source includes all the following PMHC Program Types:
+The `All PMHC Program Types` data source includes all the following PMHC Program Types:
 
 .. contents::
    :local:
    :depth: 3
 
-Data linked to Beyond Blue The Way Back extension is included in all of these Program Types.
-In order to partition reports by Way Back data a separate selection has been provided.
+Data linked to the previous Beyond Blue The Way Back extension, has been migrated to the Universal Aftercare Program Type.
 
-The PMHC data source does not include any hAPI headspace extension data.
+.. note::
+  The `All PMHC Program Types` data source does not include any hAPI headspace extension data.
 
 .. _flexible-funding-pool:
 
 Flexible Funding Pool
 +++++++++++++++++++++
 
-Other Programs commissioned through Primary Mental Health Care Schedule that are
-not otherwise described by another category. (See `PMHC Program Type <https://docs.pmhc-mds.com/projects/data-specification/en/v4/data-model-and-specifications.html#program-type>`_).
-
-.. _head-to-health:
+Programs commissioned through the Primary Mental Health Care Schedule that are
+not otherwise described by another category. (See `PMHC Program Type <https://docs.pmhc-mds.com/projects/data-specification/en/latest/data-model-and-specifications.html#program-type>`_).
 
-Head to Health
-++++++++++++++
+Head to Health Clinics
+++++++++++++++++++++++
 
-The Head to Health Program includes Head to Health Adult Centres and Satellites
-and pop-up clinics. (See `PMHC Program Type <https://docs.pmhc-mds.com/projects/data-specification/en/v4/data-model-and-specifications.html#program-type>`_).
-
-.. _amhc:
-
-AMHC
-++++
-
-The Head to Health Program that are delivering the Adult Mental Health Centre (AMHC)
-trial sites. (See `PMHC Program Type <https://docs.pmhc-mds.com/projects/data-specification/en/v4/data-model-and-specifications.html#program-type>`_).
+Mental health services delivered through Head to Health Clinics in NSW 
+and Victoria. (See `PMHC Program Type <https://docs.pmhc-mds.com/projects/data-specification/en/latest/data-model-and-specifications.html#program-type>`_).
```

Comment:
The inconsistency of state abbreviation feels awkward. Perhaps either ‘New South Wales and Victoria’ or abbreviate both.